sovtoken/sovtoken/test/helpers/helper_request.py

```diff
@@ -137,6 +147,28 @@ def schema(
         request = self._sign_sdk(request, sdk_wallet=sdk_wallet)
         return request
 
+    def acceptance_mechanism(self, sdk_trustee_wallet, aml, aml_context=None):
+        acceptance_mechanism_future = build_acceptance_mechanism_request(sdk_trustee_wallet[1], aml, aml_context)
+        acceptance_mechanism_request = self._looper.loop.run_until_complete(acceptance_mechanism_future)
+        acceptance_mechanism_request = self._sdk.sdk_json_to_request_object(json.loads(acceptance_mechanism_request))
+        acceptance_mechanism_request = self._sign_sdk(acceptance_mechanism_request, sdk_trustee_wallet)
```

Comment:
Why do we need to convert the request into Request object to pass it to sign?

Reply:
Plenum API that uses sdk wallet requires Request for some reason